reimbursement.md

```diff
@@ -232,7 +231,7 @@ curl -X POST \
   }'
 ```
 
-After the reimbursement request is received by the card scheme, the full transaction is returned with the newly created `transaction.reimbursement` object with `pending` status.
+After the reimbursement request is received by the card scheme, the full transaction is returned with the newly created `transaction.reimbursement` object with `pending` status, and also a `reimbursement.token` string, representing an unique identifier for that reimbursement request.
```

Comment:
we don't detail any field present in the `transaction.reimbursement`object. do we need to specify the token field here?

Reply:
Yes, in the ticket description, there's a AC for this - `Add a small description about the purpose of the reimbursement.token field`

Reply:
I agree that we need to explain what the token field represents. But I don't feel it belongs here. Maybe in the `Creating a request` makes more sense? Since we are explaining what the fields represent there, we could also explain that the response will include a token representing that unique reimbursement that was created?

Reply:
We can pair and try to find a better place for it. Maybe we could have a section explaining the response fields? Not sure, we have multiple options, but this one in particular I don't believe it's the best one